internal/model/db_mysql.go

```diff
@@ -0,0 +1,220 @@
+// Copyright 2021 The golang.design Initiative Authors.
+// All rights reserved. Use of this source code is governed
+// by a MIT license that can be found in the LICENSE file.
+//
+// Originally written by Mai Yang <maiyang.me>.
+
+package model
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"time"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/jmoiron/sqlx"
+)
+
+// MySQLDB MySQL DB struct
+type MySQLDB struct {
```

Comment:
```suggestion
type MySQLDB struct {
```

如果不支持多库的话，就直接命名 `Store` 或者 `DB`，这样内部使用什么数据库对外部而言也不在名字上体现